code/game/objects/items/devices/radio/headset.dm

```diff
@@ -354,7 +354,7 @@
 		set_undefibbable_on_minimap(z_level, marker_flags)
 		return
 
-	if(wearer.stat == DEAD)
+	if(wearer.stat == DEAD || wearer.chestburst == 2)
```

Comment:
Why not have bursting show them as perma dead, instead of the revivable one?

Reply:
also, wearer.chestburst == 2 isn't really clear, can you make the `2` into a define?